pkg/kubelet/cm/podresources/client.go

```diff
@@ -23,12 +23,13 @@ import (
 
 	"google.golang.org/grpc"
 
-	podresourcesapi "k8s.io/kubernetes/pkg/kubelet/apis/podresources/v1alpha1"
+	kubeletpodresourcesv1 "k8s.io/kubelet/pkg/apis/podresources/v1"
+	kubeletpodresourcesv1alpha1 "k8s.io/kubelet/pkg/apis/podresources/v1alpha1"
 	"k8s.io/kubernetes/pkg/kubelet/util"
 )
 
 // GetClient returns a client for the PodResourcesLister grpc service
```

Comment:
update comment to refer to GetV1alphaClient, and add a Deprecated: comment saying to use the GetV1Client function below.

Reply:
Fixed!